src/components/Personalization/index.js

```diff
@@ -93,6 +145,14 @@ const createPersonalization = ({ config, logger, eventManager }) => {
 
         collectClicks(merger, clickedElement, storage);
       }
+    },
+
+    commands: {
+      getDecisions(options = {}) {
+        const { scopes = [] } = options;
```

Comment:
Should the default be to get all decisions?  Currently, this will return all decisions with the default scope.  As we talked in standup, should there be a way to return default decisions and another type of decision?